README.md

```diff
@@ -25,7 +25,7 @@ Available [here](https://dotnet.microsoft.com/download)
 **2. PostgreSQL 9.6 or above database with PLV8**
 
 The fastest possible way to develop with Marten is to run PostgreSQL in a Docker container. Assuming that you have
-Docker running on your local box, type `dotnet run -- init-db` at the command line to spin up a Postgresql database with
+Docker running on your local box, type `dotnet run --framework net5.0 -- init-db` at the command line to spin up a Postgresql database with
```

Comment:
Thought this would save people a minute or two, but doesn't have to be included in this PR.

Reply:
That's fine.